src/CommandingOfficers/RuinedCommander.java

```diff
@@ -51,8 +51,8 @@ public abstract class RuinedCommander extends DeployableCommander
           + "Your COU is always max level.\n"
           + "Veterancy bonuses do not apply if this CO is tagged out, but kills are always tracked.\n"
           );
-  public static final int DOR_STAR_VALUE = 10000;
-  public static final int CHARGERATIO_HP =  1000; // Funds value of 1 HP damage dealt, for the purpose of power charge
+  public static final int DOR_STAR_VALUE = 10;
+  public static final int CHARGERATIO_HP =  1; // Funds value of 10 HP damage dealt, for the purpose of power charge
```

Comment:
All right, so DoR COs now get 1 energy per HP lost, and need 10 energy per star. Yet the energy display in the CO Info pane still shows values in the tens of thousands. How are you reconciling that with the differing power cost bases?

Reply:
Eh? It showed up as 60 for me, and I don't think I have uncommitted changes.

Reply:
No, you are right, I just forgot to switch to the new branch when I moved to the new PR.

Though it does seem a bit odd to see three orders of magnitude difference in the "Power to go" measurements for different COs. Not super sure what to do about that though. Maybe change it to display in stars so we aren't comparing the underlying nonstandard units?

Reply:
It's odd, yes, but IMO it's good for the mechanics to be obvious.
DoR charge goes "Hey look, it changed by the HP damage I just dealt", which is super obvious.

Reply:
That leaves the non-DoR four-digit magnitudes confusing though, potentially. I guess a universal "stars to go" isn't necessarily more clear either.

Reply:
Four-digit magnitudes were specifically requested, since they refer to the funds of damage directly (so you can grok them if you know what's going on).
Three-digit AWDS magnitudes are also good IMO since it is clearly neither of the other options, and that's about as clear as it's getting without adding a stats screen/page that shows the energy-per-HP stat.